myrdstom/store-manager-frontend
# Chore setup react router 164330036

**What does this PR do?**
This PR delivers a chore to Setup react router version 4 in the Author’s Haven frontend project

**Description of Task to be completed?**
- Setup react router v4 to enable server-side rendering

**How should this be manually tested?**
- Clone the repository from [Here](https://github.com/andela/Ah-frontend-guardians.git)
- Install dependencies by running `yarn install`
- Run the application with `yarn start`
- Navigate between `home` and `about` as you like. You should notice that it does not reload when switching between pages

**What are the relevant Pivotal Tracker stories?**
[#164330036](https://www.pivotaltracker.com/story/show/164330036)
